### PR TITLE
Clean up integration test

### PR DESCRIPTION
**Pull request by rkondratowicz**
Minor cleanup in preparation for getting rid off zipped files from test resources.

### CODE REVIEW

**Comment by rkondratowicz** [comment]
more to come

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTestForFailedStatus.java [hunk 1]
```diff
@@ -47,24 +45,20 @@ public void should_record_failure_of_upload_when_document_management_returns_emp
...
 
         ProcessEvent processEvent = processEvents.get(0);
-        assertThat(processEvent)
-            .extracting("container", "zipFileName", "event")
-            .hasSameElementsAs(
-                ImmutableList.of(
-                    testContainer.getName(),
-                    VALID_ZIP_FILE_WITH_CASE_NUMBER, DOC_UPLOAD_FAILURE
-                )
-            );
```
Comment (rkondratowicz) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTestForFailedStatus.java]: :O

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java [hunk 2]
```diff
@@ -78,23 +77,23 @@ public void should_read_blob_check_signature_and_save_metadata_in_database_when_
...
+        // given
+        Pdf pdf1 = new Pdf("1111001.pdf", toByteArray(getResource("1111001.pdf")));
+        Pdf pdf2 = new Pdf("1111002.pdf", toByteArray(getResource("1111002.pdf")));
 
         given(documentManagementService.uploadDocuments(ImmutableList.of(pdf1, pdf2)))
-            .willReturn(getFileUploadResponse());
+            .willReturn(ImmutableMap.of(
+                "1111001.pdf", DOCUMENT_URL1,
+                "1111002.pdf", DOCUMENT_URL2
+            ));
```
Comment (rkondratowicz) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java]: For clarity. Some tests zip file happen to contain the same pdfs.

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java [hunk 3]
```diff
@@ -78,23 +77,23 @@ public void should_read_blob_check_signature_and_save_metadata_in_database_when_
...
 
         doNothing().when(serviceBusHelper).sendMessage(any(Msg.class));
 
-        //when
+        // when
```
Comment (doncem) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java]: feels good

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java [hunk 4]
```diff
@@ -78,23 +77,23 @@ public void should_read_blob_check_signature_and_save_metadata_in_database_when_
     }
 
     private void testBlobFileProcessed() throws Exception {
-        byte[] test1PdfBytes = toByteArray(getResource("1111001.pdf"));
-        byte[] test2PdfBytes = toByteArray(getResource("1111002.pdf"));
-
-        Pdf pdf1 = new Pdf("1111001.pdf", test1PdfBytes);
-        Pdf pdf2 = new Pdf("1111002.pdf", test2PdfBytes);
+        // given
+        Pdf pdf1 = new Pdf("1111001.pdf", toByteArray(getResource("1111001.pdf")));
```
Comment (nitinprabhu) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java]: can `"1111001.pdf"` and `"1111002.pdf"`  be extracted to constant string ?

**Pull request merged**